clicache/integration-test/CacheHelperN.cs

```diff
@@ -1888,7 +1888,7 @@ public static int getBaseLocatorPort()
           extraLocatorArgs += sslArgs;
         }
 
-        string locatorArgs = LocatorStartArgs + " --name=" + serverName + startDir + extraLocatorArgs;
+        string locatorArgs = LocatorStartArgs + " --name=" + serverName + startDir + extraLocatorArgs + " --http-service-port=0";
```

Comment:
This change shaves about 10s off each test!

Reply:
Very cool!